docs/source/custom_resolvers.rst

```diff
@@ -298,6 +298,45 @@ This resolver can be useful for instance to parse environment variables:
     type: int, value: 3308
 
 
+.. _oc.select:
+
+oc.select
+^^^^^^^^^
```

Comment:
General doc comment:
There is another situation where `oc.select` can be useful which I believe is worth documenting: selecting from a missing value (which either yields `None` or the provided default -- by the way it looks like only the second case is tested).
It's an important difference with interpolations that don't allow interpolating to a missing node.